.gitignore

```diff
@@ -10,6 +10,9 @@
 /setup.h
 /*tags
 /autom4te.cache
+/.vs
+/out
```

Comment:
This isn't MSVS default output folder, so I don't know why should we ignore this one.

Reply:
Here's the content of the default CMakeSettings.json that is created when clicking Project -> CMake Settings for wxWidgets:

```json
{
  "configurations": [
    {
      "name": "x64-Debug",
      "generator": "Ninja",
      "configurationType": "Debug",
      "inheritEnvironments": [ "msvc_x64_x64" ],
      "buildRoot": "${projectDir}\\out\\build\\${name}",
      "installRoot": "${projectDir}\\out\\install\\${name}",
      "cmakeCommandArgs": "",
      "buildCommandArgs": "",
      "ctestCommandArgs": ""
    }
  ]
}
```

So `out` is indeed the default build+install dir in this case.

Reply:
Huh, sorry, I didn't know this. Where does this come from? I can't find `out` anywhere under `build/cmake`.

Reply:
It's predefined in VS: https://learn.microsoft.com/en-us/cpp/build/cmake-predefined-configuration-reference?view=msvc-170#x86-64-predefined-build-configurations